AutoRest/Generators/CSharp/Azure.CSharp.Tests/Expected/AcceptanceTests/HeadExceptions/HeadExceptionOperations.cs

```diff
@@ -141,7 +141,12 @@ internal HeadExceptionOperations(AutoRestHeadExceptionTestService client)
             if ((int)_statusCode != 200)
             {
                 var ex = new CloudException(string.Format("Operation returned an invalid status code '{0}'", _statusCode));
-                _responseContent = await _httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
+                if (_httpResponse.Content != null) {
```

Comment:
Agreed, that's my preferred way of structuring. Always a debate with myself over how efficient vs readable to make it. Since the coalesce makes sense to you in this case too, I'll use that here.